blogposts/universal-code-search-gitlab.md

```diff
@@ -0,0 +1,29 @@
+---
+title: 'Universal Code Search for GitLab'
+author: Ryan Blunden
+authorUrl: https://twitter.com/ryan_blunden
+publishDate: 2020-03-12T05:59-06:00
+tags: [
+  blog
+]
+slug: universal-code-search-gitlab
+heroImage: /external-logos/gitlab-mark.svg
+published: true
+---
+
+Sourcegraph [Universal Code Search](/universal-code-search) enables developers to explore and better understand all code, faster, with contextual code intelligence.
+
+This screencast (with accompanying audio) shows how to [connect and configure](https://docs.sourcegraph.com/admin/external_service/gitlab) Sourcegraph Universal Code Search for public and private repositories on GitLab.com and GitLab CC/EE instances
```

Comment:
```suggestion
This screencast (with accompanying audio) shows how to [connect and configure](https://docs.sourcegraph.com/admin/external_service/gitlab) Sourcegraph Universal Code Search for public and private repositories on GitLab.com and GitLab CE/EE instances
```

Reply:
Perhaps this is more clear in the screencast, but there's a big difference between enabling for my account on GitLab.com and all the other configuration/admin steps when running your own GitLab instance. Can you make sure to distinguish between the two here?

Reply:
Good call re GitLab.com vs. GitLab self-hosted. I'll change the wording so it's specific to CE/EE.